samples/compute_block_ssh_keys.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPComputeBlockSSHKeysConstraintV1
+metadata:
+  name: compute_block_ssh_keys
+  annotations:
+    description: CIS-1.1 - 4.03 Ensure "Block Project-wide SSH keys" is enabled for VM instances
```

Comment:
CIS constraints should follow the existing annotation standard.

https://github.com/GoogleCloudPlatform/policy-library/blob/master/samples/compute_forbid_ip_forward.yaml#L20